### PR TITLE
add raychat.io platform

### PR DESCRIPTION
**Pull request by samanbaboli**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -10560,7 +10560,16 @@
...
 			"website": "http://xuijs.com"
-		}
+		},
+    "Raychat": {
+      "cats": [
+        "52"
+      ],
+      "env": "^Raychat$",
+      "icon": "raychat.svg",
+      "script": "(?:app.raychat.io/scripts/js/)",
```
Comment (gadcam) [inline src/apps.json]: You don't need a group so you can remove it. I think it would be better to escape the dots with two backslashes.

Comment (samanbaboli) [reply src/apps.json]: ok it's done.

Comment (AlexByte) [reply src/apps.json]: with **two** backslashes!

Comment (samanbaboli) [reply src/apps.json]: when escape the dots with two backslashes regex not match with my script.

Comment (AlexByte) [reply src/apps.json]: How do you test it?

Comment (samanbaboli) [reply src/apps.json]: test with regextester.com

Comment (AlexByte) [reply src/apps.json]: You did not use regular expressions in programming languages?
> When using the constructor function, the normal string escape rules (preceding special characters with \ when included in a string) are necessary. For example, the following are equivalent:
> ```
> var re = /\w+/;
> var re = new RegExp('\\w+');
> ```

**Comment by jvoisin** [comment]
Please do rebase your commits on top of master, travis as been fixed \o/

**Comment by AlexByte** [comment]
@jvoisin what's the point of rebase?

**Comment by AlexByte** [comment]
Why remove the svg icon?

**Comment by samanbaboli** [comment]
The Travis CI build is error to SVG file 
as I remember was an error in a condition 
if(type == 'svg') ... 
when in convert my SVG file to png, this problem solved

**Comment by gadcam** [comment]
Travis was broken see #1722 maybe you could try again after a rebase. Your pull request with the SVG file seems good to me.

**Comment by AlexByte** [comment]
@gadcam It's not the problem with Travis (fixed in https://github.com/AliasIO/Wappalyzer/pull/1716/commits/23349db323e13ba94ed768799ebfd23be7e9acf9). Its the problem with 
https://github.com/AliasIO/Wappalyzer/blob/fdb5176af725d8700b07c6bc5b41e7793b86be0e/bin/validate-icons

**Comment by AlexByte** [comment]
@gadcam I don't understand why in [this ](https://github.com/AliasIO/Wappalyzer/pull/1731)pull-request check was successfully executed.

**Comment by gadcam** [comment]
I would try to rebase before making any other assumption.

**Comment by AlexByte** [comment]
I have made rebase https://github.com/AlexByte/Fixpalyzer/pull/16

**Comment by samanbaboli** [comment]
What should I do now?

**Comment by AlexByte** [comment]
You should reset to the commit `23349db` or rebase to upstream master and reset to the commit `aa1acab`.

**Comment by AlexByte** [comment]
Is it inconvenient to download the repository locally? More conveniently using Github to edit?

**Comment by AliasIO** [comment]
I'll sort this one out, thank you!

**Pull request merged**